static/src/javascripts/projects/common/modules/commercial/dfp/init.js

```diff
@@ -45,23 +48,34 @@ define([
             window.googletag.cmd.push(
                 setListeners,
                 setPageTargeting,
+                setAudienceScienceCallback,
                 resolve
             );
         });
     }
 
     function setListeners() {
-
         ophanTracking.trackPerformance(window.googletag);
-
-
         window.googletag.pubads().addEventListener('slotRenderEnded', raven.wrap(onSlotRender));
     }
 
     function setPageTargeting() {
+        var pubads = window.googletag.pubads();
         var targeting = buildPageTargeting();
         Object.keys(targeting).forEach(function (key) {
-            window.googletag.pubads().setTargeting(key, targeting[key]);
+            pubads.setTargeting(key, targeting[key]);
         });
     }
+
+    // Remove all Audience Science related targeting keys as soon as we recieve
+    // an AS creative (will get called by the creative itself)
+    function setAudienceScienceCallback() {
+        window.asiDirectPrequal = function () {
+            var pubads = window.googletag.pubads();
```

Comment:
could you add some tracking so that we know how many times `asiDirectPrequal` is called? Maybe using `beacon`? Also, the function name is a bit cryptic, could it be something like `audienceScienceCreativeLoaded`?